README.md

```diff
@@ -257,6 +257,20 @@ val observable =
 
 Enjoy!
 
+### Caveats
```

Comment:
Thank you @allantl 
Can we also add a link to the issue?

Reply:
Done!